### DISCUSSION TITLE
how to copy magento blank theme and modify it for a landing page in docker-based magento

### DISCUSSION THREAD

**Discussion by principezinha** [community/community#84070] [New to GitHub]
hi together, i am working on an ecological-based project and need help. I have studied sociology and cultural studies and had to learn html and css because i haven`t got the money to hire developers. Now i have the help of someone who programmes abit for me but she also has no idea of magento. So i am still very much stuck in the beginnings. i have the html, css and js stuff, the design and a multivendor extension. i only want to copy the magento blank theme to modify it. The magento installation i have is docker-based and different on my private server than in the cloud where i have web ftp access. i can use terminal and the backend but nothing works. i have to get access to the test shop via the terminal into the docker. so does anyone know the terminal code lines to copy magento blank theme and install it so i can see it in the magento backend of the store? i am so very, very grateful for any idea. thank you so much!!!

**Comment by queenofcorgis** [comment]
If you're asking how to use the piece of software that you found in a repository on GitHub, the best way to contact the maintainers of that software is to:

- Check the README for instructions on how to operate the software and pointers to documentation or troubleshooting info.
- Check the [SUPPORT file](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-support-resources-to-your-project), if one exists, for instructions on how to best contact the maintainers for support.
- Check the [CONTRIBUTING guide](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/setting-guidelines-for-repository-contributors), if one exists. Sometimes if there isn't a SUPPORT file, the CONTRIBUTING guide will give instructions on how to contact the maintainers for support.

All of these documents can be found, if they exist, in the repository where you found the software itself.

**Comment by akashdip2001** [comment]
To copy and modify the Magento Blank theme, you can follow these steps in the terminal. Ensure you have the necessary permissions, and make sure to back up your data before making significant changes.

1. **Access Docker Container:**
   - Open your terminal and navigate to the directory where your Docker Compose file is located.
   - Run the following command to access the Docker container:
     ```bash
     docker-compose exec php bash
     ```

2. **Navigate to Magento Root:**
   - Navigate to the Magento root directory:
     ```bash
     cd /var/www/html
     ```

3. **Copy the Blank Theme:**
   - Run the following command to copy the Blank theme:
     ```bash
     bin/magento theme:copy --from=Magento/blank --to=YourVendor/your_theme
     ```
     Replace `YourVendor` with your vendor name (e.g., `MyVendor`) and `your_theme` with your theme name.

4. **Install the Theme:**
   - Run the following command to install the new theme:
     ```bash
     bin/magento setup:upgrade
     ```

5. **Deploy Static Content:**
   - Deploy the static content by running:
     ```bash
     bin/magento setup:static-content:deploy
     ```

6. **Flush Cache:**
   - Flush the cache for the changes to take effect:
     ```bash
     bin/magento cache:flush
     ```

Now, you should be able to see your new theme in the Magento backend.

Remember to replace placeholders with your actual values, and ensure that you have the necessary backups in case anything goes wrong. If you encounter any issues, check the Magento logs (`var/log` directory) for error messages that can help diagnose the problem.

## Profile
![Profile Image](https://thumbnails.odycdn.com/optimize/s:200:0/quality:95/plain/https://thumbs.odycdn.com/bf58cf4d7ec52c62f02c52e16f150a5b.webp)
**AKASHDIP MAHAPATRA**
"everything is physics bro"  
E=mc²

**Reply by queenofcorgis** [reply]
@akashdip2001 
We’ve clarified our stance on using generative AI tools like ChatGPT within our Community via [this announcement](https://github.com/orgs/community/discussions/69006). Please review the guidelines to ensure your post meets them as failure to adhere to those rules can result in action taken by our moderator team.  You can read our updated [Code of Conduct](https://github.com/community/community/blob/baa5cd9020ef3f2bbf86dca1191d9c90f9b0822d/CODE_OF_CONDUCT.md#reasonable-use-of-ai-generated-content) and the [announcement](https://github.com/orgs/community/discussions/69006) for more details. 

Additionally, thanks for sharing your YouTube channel with the community. Unfortunately, we currently do not allow self-promotion or advertising on the Community on Discussions - we kindly ask you to edit your comments to remove your channel's link. 

Thank you for helping us maintain an authentic and beneficial space for everyone.

**Reply by akashdip2001** [reply]
Sorry about that, to add my youtube link. I just help people and i also re-check my every comment, So that no wrong information is spread :)

**Reply by queenofcorgis** [reply]
Thank you @akashdip2001 and thanks for helping out!
